app/services/participant_validator.rb

```diff
@@ -9,58 +9,19 @@ def initialize(trn:, full_name:, date_of_birth:, national_insurance_number: nil)
   end
 
   def call
-    if Feature.ecf_api_disabled?
-      call_with_dqt
-    else
-      call_with_ecf
+    result = Dqt::RecordCheck.new(**payload.merge(check_first_name_only: true)).call
+    if result.total_matched >= 3
```

Comment:
Why is 3 the magic number? 🤔 is it worth having this as a constant to help explain?

Reply:
The mystical number that came from the ancient times of ecf lol